test/smoke_test/bcda_client.go

```diff
@@ -84,6 +85,26 @@ func main() {
 	}
 }
 
+func NewClient(accessToken string, retries int) *client {
```

Comment:
Can we add a summary on the types of errors that will be retried based on this policy?